directory/src/main/java/org/entcore/directory/services/impl/DefaultUserService.java

```diff
@@ -525,31 +524,74 @@ public void listAdmin(String structureId, boolean includeSubStructure,
 						  final UserInfos userInfos,
 						  final Handler<Either<String, JsonArray>> results) {
 		JsonObject params = new JsonObject();
-		String filter = "";
-		String filterProfile = "WHERE 1=1 ";
-		String optionalMatch =
-			"OPTIONAL MATCH u-[:IN]->(:ProfileGroup)-[:DEPENDS]->(class:Class)-[:BELONGS]->(s) " +
-			"OPTIONAL MATCH u-[:RELATED]->(parent: User) " +
-			"OPTIONAL MATCH (child: User)-[:RELATED]->u " +
-			"OPTIONAL MATCH u-[rf:HAS_FUNCTION]->fg-[:CONTAINS_FUNCTION*0..1]->(f:Function) " +
-			"OPTIONAL MATCH u-[:TEACHES]->(sub:Subject) ";
-		if (expectedProfiles != null && expectedProfiles.size() > 0) {
-			filterProfile += "AND head(u.profiles) IN {expectedProfiles} ";
-			params.put("expectedProfiles", expectedProfiles);
-		}
+		boolean restrictResultsToFunction = false;
```

Comment:
Est-ce que tu pourrais ajouter ici une petite ligne de commentaire pour dire à quoi il servira, genre 
`// Used to determine if we need to limit the results to the current user's scope to avoid leaking unsupervised users` ou qqch du style ?